### DISCUSSION TITLE
2023-09-01

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#456] [daily_meeting]
### 오늘의 할 일
- 개인 일정
  - 룩소: 근로 회의가 있을 수도 있고 없을 수도 있다
  - 노아: 5시부터 근로 

- 백엔드
  - ~오전에 강의~

- 공통
  - ~솔라에게 피드백 받은거 답변 남기기~

- 프론트엔드
  - ~체크리스트 피드백 제출(각자 플로우 뭐 테스트할지 생각해오기)~
  - Auth 로직 변경(함께) - 오전
  - 각자 api 수정 - 오후 ~ 주말

### 내일의 할 일
- 사용자 유치 계획 세우기
- SSE 관련 조사해오기
- 프론트엔드
  - Auth 로직 변경 (미룸)
  - 각자 api 수정 (주말)

**Comment by woosung1223** [comment]
### 체크인
9점 - 졸리지만 금요일이라 기분이 좋을 뻔 했지만 강의가 있어서 -1점

### 체크아웃
9점 - 여전히 졸리지만.. 그래도 금요일이라 좋네요!!

**Comment by aak2075** [comment]
### 체크인

10점 - 벌써 금요일이네요

### 체크아웃

10점 - 주말을 기대하지 않아요

**Comment by nlom0218** [comment]
### 체크인

9점 - 시간이 조금더 천천히 갔으면 😭

### 체크아웃

10점 - 주말이기도 하고 SSE라는 새로운 것을 배울 수 있어서 좋았어요 :)

**Comment by yeopto** [comment]
### 체크인

8점 - 인생이 쉽지 않네요.

### 체크아웃

9점 - 주말동안 잡생각들을 정리한다고 다짐하니 한결 나아지네요

**Comment by woo-jk** [comment]
### 체크인
9점 - 금요일이라 굿

### 체크아웃
9점 - http 리팩토링 끝 ㅎ

**Comment by jaehee329** [comment]
### 체크인
8점 - 어제보다 많이 잤어요
### 체크아웃
1점 - 테코톡 언제준비함?

**Reply by nlom0218** [reply]
잠은 죽어서 자면 되는거 아닌가요?

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 어제 일찍 잔다고 했는데 역대급으로 피곤했었네요... 하지만 금요일이기 때문에 텐션 업 해보죠!

### 체크아웃
10점 - 주말에 잘 쉬면서 미션하면 될 것 같습니다. 오랜만에 미션하니까 재밌네용 ㅎㅎ
